vignettes/ScPCAr.Rmd.orig

```diff
@@ -0,0 +1,313 @@
+---
+title: "Using the ScPCAr package to access Single-cell Pediatric Cancer Atlas data"
+output: html_document
+vignette: >
+  %\VignetteIndexEntry{Using the ScPCAr package to access Single-cell Pediatric Cancer Atlas data}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# Introduction
+
+The ScPCAr package provides an interface to interact with the Single-cell Pediatric Cancer Atlas (ScPCA) Portal API.
+This vignette demonstrates the basic workflow for discovering, downloading, and working with ScPCA data.
+
+This vignette covers the following tasks for interacting with the ScPCA Portal:
+
+1. Listing available projects
```

Comment:
I was going to say a toc would be good, but then again we're going to essentially get this for free with `pkgdown`, so this seems fine.